### PR TITLE
Task update mumukit

### PR DESCRIPTION
**Pull request by flbulgarelli**

### CODE REVIEW

File: lib/plunit.rb [hunk 1]
```diff
@@ -3,4 +3,5 @@
 require_relative 'test_runner'
 require_relative 'test_compiler'
 require_relative 'expectations_runner'
+require_relative 'query_runner'
```
Comment (faloi) [inline lib/plunit.rb]: Why is this necessary?

Comment (flbulgarelli) [inline lib/plunit.rb]: Because it wasn't processable by previous version. Now it is.
